### PR TITLE
Bump dependency versions

### PR DESCRIPTION
**Pull request by Lencerf**
Update vsock from 0.2.4 to 0.3.0. Changes are made according to https://github.com/rust-vsock/vsock-rs/commit/db589aa30ce36826198aa63df8e81d7a175f5266.

### CODE REVIEW

**Review by qwandor — APPROVED** [review]

**Comment by qwandor** [comment]
Looks good, thanks! You'll just need to edit your commit message to fix the presubmit check.

**Comment by Lencerf** [comment]
Done. But maintainer approval is required to start presubmit checks.

**Comment by Tim-Zhang** [comment]
Since the changed api we need a major version release after this pr merged

**Review by Tim-Zhang — APPROVED** [review]

**Pull request merged**
